src/banking/sheba_table.rs

```diff
@@ -1,240 +1,379 @@
 use crate::utils::{create_fixed_map, FixedMap};
 
-#[derive(Debug)]
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct ShebaAccountNumber {
+    pub normal: String,
+    pub formatted: String,
+}
+
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub struct ShebaInfo {
     pub code: &'static str,
     pub nickname: &'static str,
     pub name: &'static str,
     pub persian_name: &'static str,
+    pub account_number: Option<ShebaAccountNumber>,
+    pub process: Option<fn(&mut ShebaInfo, &str)>,
```

Comment:
I'm on mobile... I cannot check it carefully... But why do we need this `process`?
To process what to what?

Reply:
So should we name this pub closure something else? Like getAccountNumber or whatever to show the main functionality?

Reply:
Oh seriously they have it with this name?
Ok, then no problem 😅 better follow their standard than inventing a new one 🙄